include/psf_utils.h

```diff
@@ -65,12 +65,17 @@ namespace psf
 
 	namespace details
 	{
+		static const unsigned int PACKAGE_PATH_TYPE_EFFECTIVE = 2;
+
+		/**
+			Summary: Used to call a method in appmodel.h and formats the return string.

```

Comment:
nit:spacing is still an issue all throughout
Use inline comments to be consistent throughout the source.